cylc/doc/etc/tutorial/runtime-introduction/.validate

```diff
@@ -21,7 +21,6 @@ set -eux
 TEMPORARYDIRECTORY=$(mktemp -dp "$HOME"/cylc-run/)
 cp -r ./* "$TEMPORARYDIRECTORY"
 cd "$TEMPORARYDIRECTORY"
-sed -i '1s;^;[scheduler]\n    abort if any task fails = True\n;' flow.cylc
```

Comment:
Yuk 😆 A lot nicer now

Reply:
It took me a while and some panicky message to Oliver to sort this. It's an important lesson with Sed though: Separators can be chars not="/", and it's often convenient if they are something else